frontend/src/i18n/locales/en-sg.ts

```diff
@@ -0,0 +1,11 @@
+import Translation from './translation.interface'
+
+const enSG: Translation = {
```

Comment:
supernit: prefer  `export const` over `export default`, but am ok! (rationale is in case we rename things, but also cognisant that translation files probably will never be renamed haha)

Reply:
done